cluster/src/main/java/org/apache/iotdb/cluster/query/reader/mult/AbstractMultPointReader.java

```diff
@@ -24,11 +24,26 @@
 import java.io.IOException;
 import java.util.Set;
 
-public interface IMultPointReader extends IPointReader {
+public abstract class AbstractMultPointReader implements IPointReader {
 
-  boolean hasNextTimeValuePair(String fullPath) throws IOException;
+  public abstract boolean hasNextTimeValuePair(String fullPath) throws IOException;
 
-  TimeValuePair nextTimeValuePair(String fullPath) throws IOException;
+  public abstract TimeValuePair nextTimeValuePair(String fullPath) throws IOException;
 
-  Set<String> getAllPaths();
+  public abstract Set<String> getAllPaths();
+
+  @Override
+  public boolean hasNextTimeValuePair() throws IOException {
+    return false;
+  }
+
+  @Override
+  public TimeValuePair nextTimeValuePair() throws IOException {
+    return null;
+  }
+
+  @Override
+  public TimeValuePair currentTimeValuePair() throws IOException {
+    return null;
+  }
```

Comment:
These methods had better be protected with exceptions.

Reply:
Thanks, done.